Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Services/UsersController.cs

```diff
@@ -69,9 +69,16 @@ public HttpResponseMessage CreateUser(CreateUserContract contract)
                     : null);
             }
             catch (Exception ex)
-            {
-                Logger.Error(ex);
-                return this.Request.CreateErrorResponse(HttpStatusCode.InternalServerError, ex.Message);
+            {
+                if (ex.GetType() == typeof(InvalidUserRegisterException))
+                {
+                    return this.Request.CreateErrorResponse(HttpStatusCode.BadRequest, ex.Message);

```

Comment:
This is a nice differentation between "expected" and "unexpected", but I do not want to change the behavior of logging the failures under ALL circumstances.  If you move the `Logger.Error(ex)` to be called all of the time I can agree with this.

Reply:
Do you want to log invalid user register attempts? I can add this as well..

Reply:
done